docs/setup/native/index.md

```diff
@@ -9,7 +9,7 @@ You'll also need a copy of the bot. We would recommend going to our [releases pa
 <summary>Advanced use: building the bot from source</summary>
 
 For advanced users, you can also download a source archive or clone the repository with `git clone https://github.com/fluid-queue/fluid-queue`. Our prepackaged releases are already bundled and minified, so these methods will require a bit of extra effort (and installation of a substantial `node_modules` folder). We'd recommend using the prepackaged releases for greater simplicity, though we will still support building from source.
-
+The [next step](./setup/) will explain how to compile the bot.
```

Comment:
This should link to `setup.md` rather than `setup/`; this ensures it'll be compiled correctly by Docusaurus, even after being versioned. When Docusaurus compiles the site, it translates links, and the recommended practice is to link to the document itself to make sure the translation can work properly.

```suggestion
The [next step](setup.md) will explain how to compile the bot.
```

Reply:
I was not aware of how that worked! Thank you for the explanation.